README-source.md

```diff
@@ -633,6 +633,41 @@ module.exports = {
 };
 ```
 
+### `bundle.rawRequest`
+***`bundle.rawRequest` is only available in the `perform` for web hooks and `getAccessToken` for oauth authentication methods***
+
+`bundle.rawRequest` holds raw information about the HTTP request that triggered the `perform` method or that represents the users browser request that triggered the `getAccessToken` call:
+
+```
+{
+	method: 'POST',
+	querystring:'key=value;key2=value2',
+	content:'{key:value}'
+}
+```
```

Comment:
Since this is Javascript objects, I'd do something more complete and neater like:

```javascript
{
  method: 'POST',
  querystring: 'foo=bar&baz=qux',
  headers: {
    'Content-Type': 'application/json'
  },
  content: '{"hello": "world"}'
}
```

Reply:
lazy me. Will fix